Add Chameleon-supported appliances

diurnalist opened the pull request:
Leaving this here, maybe it makes sense to have these in this repo, maybe not.

Commit:
Add NFS complex appliance

Commit:
Add Hello World complex appliance

priteau commented on hello-world/hello-world.yaml:
```diff
@@ -0,0 +1,70 @@
+# This describes what is deployed by this template.
+description: NFS server and client deployed with Heat on Chameleon
```
Maybe we should update the description and remove the NFS bits from hello-world appliance, since we have a separate proper NFS appliance.

priteau commented:
@diurnalist I think we had a misunderstanding about the "Hello World" appliance: I was suggesting to simplify it by removing the parts that are specific to NFS (maybe just launch some instances using an ResourceGroup), not removing the template entirely.

diurnalist commented:
@priteau ah yes, I see what you mean now. I'll bring it back in a simpler format.

Commit:
Update Hello World to not use so much NFS

Instead, strip down the example to bare-bones, just altering some ssdh
configuration.

Commit:
Add instance_count to readme

diurnalist commented:
@priteau I have updated the Hello World appliance to use a new (very contrived!) example for demonstration purposes.

priteau reviewed (approved):
Looks good!

zhenz commented:
@diurnalist Do we have an auto-sync mechanism between this repo and Appliance Catalog? or do we have to manually do the sync?

diurnalist commented:
@zhenz there is no automatic sync as of yet.
